From 32397ea936145f82d7072ffa75452357c9c0aff0 Mon Sep 17 00:00:00 2001
From: Jiri Benc <jbenc@redhat.com>
Date: Fri, 12 Sep 2014 14:30:20 -0400
Subject: [net] openvswitch: fix duplicate #include headers

Message-id: <281e0ff70f7b9146c17323ce89794d2a994cb610.1410531783.git.jbenc@redhat.com>
Patchwork-id: 94008
O-Subject: [RHEL7.1 net PATCH 47/50] openvswitch: fix duplicate #include headers
Bugzilla: 1110384
RH-Acked-by: Flavio Leitner <fbl@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110384
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7948291

commit 2072ec846a4c4ee99a6e59ce989b49b22edad59d
Author: Jean Sacren <sakiwit@gmail.com>
Date:   Thu Aug 7 06:50:44 2014 -0600

    openvswitch: fix duplicate #include headers

    The #include headers net/genetlink.h and linux/genetlink.h both were
    included twice, so delete each of the duplicate.

    Signed-off-by: Jean Sacren <sakiwit@gmail.com>
    Cc: Pravin Shelar <pshelar@nicira.com>
    Cc: dev@openvswitch.org
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/openvswitch/datapath.c b/net/openvswitch/datapath.c
index db8830e..2da06e8 100644
--- a/net/openvswitch/datapath.c
+++ b/net/openvswitch/datapath.c
@@ -47,8 +47,6 @@
 #include <linux/openvswitch.h>
 #include <linux/rculist.h>
 #include <linux/dmi.h>
-#include <linux/genetlink.h>
-#include <net/genetlink.h>
 #include <net/genetlink.h>
 #include <net/net_namespace.h>
 #include <net/netns/generic.h>
-- 
1.7.1